conf/ocsci/default_config.yaml

```diff
@@ -37,12 +37,14 @@ RUN:
   kubeconfig_location: 'auth/kubeconfig' # relative from cluster_dir
   cli_params: {}  # this will be filled with CLI parameters data
   client_version: '{{ DEPLOYMENT["installer_version"] }}'
+  force_download_client: True
   bin_dir: './bin'
 
 # In this section we are storing all deployment related configuration but not
 # the environment related data as those are defined in ENV_DATA section.
 DEPLOYMENT:
   installer_version: "4.1.0-rc.5"
```

Comment:
I saw the previous comment about the version but any chance to change this to latest and that's it?
By that we are sure to get the latest downstream which is what we want right?

Reply:
@RazTamir it is not directly possible, because the files in `latest` [directory](https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/) still have particular version in the name, so there will have to be some additional code, which will parse the index page and chose the proper version...
So it is definitely worth to consider it, but I would propose to create new PR for that functionality.
Also I'm not sure, how the `latest` link works, because now (2019-06-11 14:45 CEST), it points to version 4.1.0 from June 5, but there is already newer version `4.1.1` from today...